### DISCUSSION TITLE
CORS origin issue on pimcore 10.6.9

### DISCUSSION THREAD

**Discussion by Thejaswini-Rao-U** [pimcore/pimcore#17361] [Q&A / Support]
### Pimcore version

10.6.9

### Steps to reproduce

Hi 
When im importing data arround 3k plus, im getting error of cross origin issue attached below

![edited](https://github.com/user-attachments/assets/360bbabb-cb8b-4f48-9e04-bff7f728edd8)

in the controller where the import method is calling, i have added the below code 
$http_origin = $_SERVER['HTTP_ORIGIN'];
        header("Access-Control-Allow-Origin:".$http_origin);
but this is not working.

In server like prod, QA import doesn't have issue if we import from Linux OS but in windows OS getting same cross origin issue

Please let me know how to solve this issue

### Actual Behavior

Not working

### Expected Behavior

Should work

**Comment by kingjia90** [comment]
Hi,

What's the bundle you are using? I don't recognize this import tool i see in the screenshot's background, at first glance looks like it may be caused there

TIA

**Comment by Thejaswini-Rao-U** [comment]
@kingjia90 , we are using normal import using php code, we dont use any plugin

**Reply by kingjia90** [reply]
Unfortunately, i think it's going outside of the area which we can intervene. 

You may have to check if the domain is not cross origin, by making sure the frame url is in the same protocol http/https, hostname and even port. If it's intended to be different, maybe need some other approach, like by a `curl`.

In my past personal experiences, i have never been able to "bypass" this by adding the header `Access-Control-Allow-Origin` or using browser extensions to allow CORS etc.. the only way is to stick to the rules/policy or lowering the browser security (which is not recommended and unpractical for end users in production anyways).

Hope it helps

**Comment by fashxp** [comment]
moving this to discussions, 10.6 is EOL.
